doc/modules/model_evaluation.rst

```diff
@@ -830,7 +830,7 @@ follows.
      <http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.157.5766&rep=rep1&type=pdf>`_,
      IJCV 2010.
   .. [Davis2006] J. Davis, M. Goadrich, `The Relationship Between Precision-Recall and ROC Curves
-     <http://www.machinelearning.org/proceedings/icml2006/030_The_Relationship_Bet.pdf>`_,
+     <https://www.researchgate.net/profile/Mark-Goadrich/publication/215721831_The_Relationship_Between_Precision-Recall_and_ROC_Curves/links/57918c1808ae64311c11ac64/The-Relationship-Between-Precision-Recall-and-ROC-Curves.pdf>`_,
```

Comment:
Could use the following:

https://www.biostat.wisc.edu/~page/rocpr.pdf

The URL is shorter this might be better

Reply:
Happy to make the update, I thought RG might be more stable long term though. Just say the word!

Reply:
It is a bit easier for readability when getting the documentation via the docstring :)
I agree that RG might sometimes be more stable (even though I got some articles removed for copyright issues and needed to add back the preprint)

Reply:
Good points, updated! ✅